_style_guide/typography-code-block.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Code Block
+type: typography
+---
+
+```crystal
```

Comment:
I suggest to add what the block shows (same for the rest of the code blocks below):

```suggestion
Crystal code:
```crystal
```

Reply:
I don't think this is necessary. The language should be obvious and is clearly visible in the source code.

Reply:
That's true now that you show the md, smarty-pants ;-)